kofta/public/locales/gsw/translation.json

```diff
@@ -63,43 +63,43 @@
       "voiceSettings": "Sprachistellige",
       "soundSettings": "Tonistellige",
       "deleteAccount": "Konto lösche",
-      "overlaySettings": "go to overlay settings"
+      "overlaySettings": "Gang zu de Overlay Istellige"
     },
     "notFound": {
-      "whoopsError": "Hoppla! Die Site isch im Gspräch vrlore gange.",
-      "goHomeMessage": "Mach dr ke Sorge. Du chasch",
-      "goHomeLinkText": "hei gah"
+      "whoopsError": "Hoppla! Die Site isch im Gspräch verlore gange.",
+      "goHomeMessage": "Mach dir kei Sorge. Du chasch",
+      "goHomeLinkText": "Hei gah"
     },
     "room": {
       "speakers": "Sprächer",
       "requestingToSpeak": "Gsprächsafrage",
       "listeners": "Zuehörer",
-      "allowAll": "Allow all",
-      "allowAllConfirm": "Are you sure? This will allow all {{count}} requesting users to speak"
+      "allowAll": "Alles erlaube",
+      "allowAllConfirm": "Bisch der sicher? Das wird all {{count}} uffordere zum rede"
```

Comment:
watch out this is a wrong translation, correct one would be: "Bisch der sicher? Das wird all {{count}} Benutzer wo wennd rede Berechtigung gäh zum rede"